app/decorators/application_choice_export_decorator.rb

```diff
@@ -0,0 +1,47 @@
+class ApplicationChoiceExportDecorator < SimpleDelegator
+  def gcse_qualifications_summary
+    gcses = %i[maths_gcse english_gcse science_gcse]
+    summary_string = gcses
+      .map { |gcse| application_form.send(gcse) }
+      .map { |gcse| summary_for_gcse(gcse) }
+      .compact
+      .join(',')
+
+    summary_string.presence
+  end
+
+  def missing_gcses_explanation(separator_string: ',')
+    application_form
+      .application_qualifications
+      .select { |q| q.level == 'gcse' }
```

Comment:
We could presumably do this with ActiveRecord, though you might have to mitigate lazy loading.